check-links/crawl-for-broken-links.sh

```diff
@@ -34,13 +33,15 @@ www\.reddit\.com/r/openziti
 .*\.home
 .*\.lan
 twitter\.com/(OpenZiggy|OpenZiti)
+x\.com/(OpenZiggy|OpenZiti)
 landing.openziti.io/
 fonts.gstatic.com/
 github\.com/.*#
 .*\.?example\.(com|net|org)
 openziti.io(/comments)?/feed/
 developer.chrome.com/origintrials/#/register_trial/.*
 entra\.microsoft\.com
+github\.com/openziti/ziti-tunnel-sdk-c%3E
```

Comment:
This is unexpected because the crawler will only try to follow hyperlinks, and links don't usually contain `>` characters.